scripts/merge_all.py

```diff
@@ -5,18 +5,45 @@
 from os import path as pt
 import re
 from collections import defaultdict
-import sys
+import sys, getopt
 
-if len(sys.argv) != 3:
-    print("Usage: {} <CROW_HEADERS_DIRECTORY_PATH> <CROW_OUTPUT_HEADER_PATH>".format(sys.argv[0]))
+if len(sys.argv) < 3:
+    print("Usage: {} <CROW_HEADERS_DIRECTORY_PATH> <CROW_OUTPUT_HEADER_PATH> <-i(include) OR -e(exclude) item1,item2...>".format(sys.argv[0]))
```

Comment:
Maybe add info about include/exclude in `[]` or `()` to show that this is optional?

Reply:
I thought since they were options rather than arguments that they're optional by default